server/repository/src/db_diesel/currency_row.rs

```diff
@@ -0,0 +1,75 @@
+use super::{currency_row::currency::dsl as currency_dsl, StorageConnection};
+
+use crate::repository_error::RepositoryError;
+
+use chrono::NaiveDate;
+use diesel::prelude::*;
+
+table! {
+    currency (id) {
+        id -> Text,
+        rate -> Double,
+        currency_code -> Text,
```

Comment:
Seems like `currency` in central saves currency codes, so have renamed it (and also column can't be same as table name, will still be something similar to currency_code even with `sql_name`)

Reply:
good call! this could even be just `code`